include/mata/nfa.hh

```diff
@@ -537,18 +552,18 @@ struct Nfa
     //  dictionary in the attributes.
     std::unordered_map<std::string, void*> attributes{};
 
-    size_t max_state_;
+    size_t m_num_of_states;
 
 public:
-    Nfa() : delta(), initial(), final() {}
+    Nfa() : delta(), initial(), final(), m_num_of_states(0) {}
 
     /**
      * @brief Construct a new explicit NFA with num_of_states states and optionally set initial and final states.
      */
     explicit Nfa(const unsigned long num_of_states, const StateSet& initial_states = StateSet{},
                  const StateSet& final_states = StateSet{}, Alphabet* alphabet_p = new IntAlphabet())
-        : delta(num_of_states), initial(initial_states), final(final_states),
-          alphabet(alphabet_p), max_state_(0) {}
+        : delta(num_of_states), initial(initial_states), final(final_states), alphabet(alphabet_p),
+          m_num_of_states(0) {}
```

Comment:
Can we merge this with previous line, so we have one liner initialization?

Reply:
I will fix that. This must have been autoformatted.

Reply:
Yeah, you have probably limit `80`, which is sooo obsolete.

Reply:
I use limit 120 and this one line initializer list is 121 or 122. Hence, the autoformat have taken over.

Reply:
I see, then maybe shorten `alphabet_p` to just `alph_p` to stay in `120` limit.

Reply:
That could be done, but I do not think it is necessary. I do not run autoformat on Mata as there are way too many different styles and inconsistencies. The autoformat would rewrite half the library. I can accidentally run the autoformat here and there on a line selection, though. I believe this was the case and I just did not notice in the final check before the commit.

I consider this one line, even if one or two characters longer than 120 characters, acceptable and readable.

Reply:
Sure, I have no strong feelings towards that :). Just that maybe some linters would write warnings about too long lines (my experience).

Reply:
That is a good point. There is no need to introduce new linter warnings when we know about them. I will rename some parameter name to fit the whole line into the limit.

Reply:
FYI: I'm not sure what is the limit in linters. I think it might be 120, but it might be 100 and this would be pointless :).

Reply:
The limit is whatever you set as your project default. We do not have any setup in Mata, so this is mostly for our IDEs to not complain. Hopefully, everyone will be able to set their IDEs to respect the 120 characters limit.